src/game/game.c

```diff
@@ -164,8 +162,8 @@ void CreateGameStage(void)
     gUnknown_03005424 |= 0x21;
     gUnknown_03005424 &= ~0x80;
 
-    gUnknown_03005430 = 0;
-    gUnknown_0300542C = 10;
+    gBossRingsShallRespawn = FALSE;
+    gBossRingsRespawnCount = BOSS_RINGS_DEFAULT_RESPAWN_COUNT;
```

Comment:
oooo

Reply:
This is the amount of times *all* rings spawn btw. (in a boss stage)
Couldn't think of a better name!

Reply:
And I'm not 100% sure whether it's limited to bosses, since it seems to be used in the Ring-Collect mode